make/test.mk

```diff
@@ -145,6 +147,8 @@ deploy-member:
 	oc new-project $(MEMBER_NS)
 	oc apply -f /tmp/member-operator/deploy/service_account.yaml
 	oc apply -f /tmp/member-operator/deploy/role.yaml
-	cat /tmp/member-operator/deploy/role_binding.yaml | sed s/\REPLACE_NAMESPACE/$(MEMBER_NS)/ | oc apply -f -
+	oc apply -f /tmp/member-operator/deploy/role_binding.yaml
+	oc apply -f /tmp/member-operator/deploy/cluster_role.yaml
```

Comment:
I hope, you are going to create a pr for this in member-operator as well, otherwise this deployment will fail

Reply:
here you go: https://github.com/codeready-toolchain/member-operator/pull/56 ;-)